app/dto/CreateCategory.ts

```diff
@@ -16,7 +16,7 @@ class CreateCategoryDto {
   @MinLength(10, {
     message: 'Description is too short'
   })
-  @MaxLength(50, {
+  @MaxLength(80, {
```

Comment:
Maybe put this value somewhere as variable so you don't have to remember to update in `UpdateCategory.ts` also :D